tests/Operation/WatchFunctionalTest.php

```diff
@@ -536,6 +535,25 @@ public function provideTypeMapOptionsAndExpectedChangeDocument()
         ];
     }
 
+    public function testNextAdvancesKey()
+    {
+        $operation = new Watch($this->manager, $this->getDatabaseName(), $this->getCollectionName(), [], $this->defaultOptions);
+        $changeStream = $operation->execute($this->getPrimaryServer());
+
+        /* Note: we intentionally do not start iteration with rewind() to ensure
+         * that we test extraction functionality within next(). */
```

Comment:
I'd suggest replacing "extraction functionality within next()" with "incrementing when first advancing within next()" or simply deleting this comment entirely.